runtime/runenv_events.go

```diff
@@ -120,6 +121,10 @@ func (re *RunEnv) RecordSuccess() {
 	}
 	re.logger.Info("", zap.Object("event", evt))
 	re.metrics.recordEvent(&evt)
+
+	if re.signalEventer != nil {
+		_ = re.signalEventer.SignalEvent(context.Background(), NewOutcomeOKNotification(re.RunParams.TestGroupID))
+	}
```

Comment:
Don't we want to publish events for all event types?

Reply:
Right. I think we should go full steam with this. Implementing it all the way is very little effort with a lot of gain.